doc/source/serve/advanced-guides/deployment-graphs.md

```diff
@@ -1,9 +1,9 @@
 (serve-deployment-graphs)=
 
-# Experimental Deployment Graphs
+# Deployment Graphs (Deprecated)
 
 :::{note}
-The call graph is in **alpha**, so its APIs are subject to change.
+This API is **deprecated** and will be removed in an upcoming release.
```

Comment:
should have a CTA -- can you link to the model composition docs and say that's the recommended pattern?